express-api/.dockerignore

```
@@ -0,0 +1,9 @@
+coverage/
+./dist
```

Just to make it look a little more like lines 1 and 3
```suggestion
dist/
```

---

Included suggested change in the code.